ex1/ex1-index.js

```diff
@@ -0,0 +1,17 @@
+function isDivisibleByTen(number) {
+    // Check if the input number is an integer
+    if (!Number.isInteger(number)) {
+        return "Error: Input must be an integer.";
+    }
+
+    // Check if the number is divisible by 10
+    if (number % 10 === 0) {
```

Comment:
```suggestion
    return number % 10;
```

se poate simplu de returnat astfel

Reply:
Super,o sa incerc sa tin cont de ele pe viitor si sa le adaug  🥂